docs/how-it-works/index.md

```diff
@@ -29,7 +29,7 @@ Projects’s AWS infrastructure.
 
 Reference AWS Cloud Solutions architecture designed under optimal configs for the most
 popular modern web and mobile applications needs based on AWS [“Well Architected Framework”](../work-with-us/support.md).
-With it's complete [**Leverage DevOps Automation Code Library**](./code-library/code-library.md) to rapidly implement it
+With it's complete [**Infrastructure as Code (IaC) Library**](./code-library/code-library.md) to rapidly implement it
```

Comment:
I think it should say "With its complete" or maybe even "With a complete" (although honestly I think that sounds a lot like the way we speak in Argentina, can't remember expressions like that in English).

Reply:
@diego-ojeda-binbash OJ may be we could use "With its full". What do you think? Do you have any other alternatives or rephrasing that could fit here?

Reply:
@exequielrafaela Can't think of anything 😅 . For now how about fixing the "it's" and later on we can think of alternatives? 😄